src/prop/sat_solver.h

```diff
@@ -98,7 +101,13 @@ class SatSolver {
   virtual bool ok() const = 0;
   
   virtual void setProofLog( BitVectorProof * bvp ) {}
-  
+
+  /** Get the top level unit clauses of a formula **/
```

Comment:
Please describe the contract in greater detail. Specifically, describe what kind of literals we can get back (only the ones added by the user? Different ones?)